websites/docs-smoke-test/src/content/release-notes.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Docs Kit Release Notes Test
+# TODO frontmatter to instruct that this is a release note layout? or MDX imports?
+---
+
+<Subtitle>Custom content above the Release notes is needed for per site customization</Subtitle>
+
+To stay informed about the latest releases subscribe to the RSS feed (TODO RSS would be better e.g. in the top right corner?) or sign up for the (test product newsletter)[https://example.com].
+
+<Warn>TODO how to put it here? Can this automatically trigger the filter in the sidebar or would we have to explicitly instruct the layout via frontmatter to not render the index nav??</Warn>
+
+`<ReleaseNotes />`
```

Comment:
Can you explain what this does?

Reply:
This was intended to trigger a conversation with you how we want authors to tell the implementation that this page is a release notes page. 

So it worked :-) - let's talk about it directly.